styles/branchingQuestion.css

```diff
@@ -58,7 +58,7 @@
   font-weight: 200;
 }
 
-.h5p-branching-scenario .h5p-branching-question button:not(.h5p-branching-question-alternative) {
+.h5p-branching-scenario .h5p-branching-question button:not(.h5p-branching-question-alternative):not(.h5p-branching-question-close) {
```

Comment:
You should probably do the below instead:
`:not(.h5p-branching-question-alternative, .h5p-branching-question-close)`